labelbox/schema/project.py

```diff
@@ -94,6 +97,15 @@ def update(self, **kwargs):
         if mode:
             self._update_queue_mode(mode)
 
+        media_type = kwargs.get("media_type")
```

Comment:
jw for the future, will we be mandating media type for project creation in the future? i see it as optional right now